dev-python/wxpython/wxpython-4.0.7-r2.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+WX_GTK_VER="3.0-gtk3"
+
+DISTUTILS_IN_SOURCE_BUILD="1"
```

Comment:
Perhaps we can also try the new PEP517 mode, i.e. `DISTUTILS_USE_PEP517=setuptools`

Reply:
let's try...